client/src/components/Review.jsx

```diff
@@ -1,6 +1,134 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components'
 
+const OrangeStar = styled.span`
+  color: orange;
+`;
+const Star = styled.span`
+  color: black;
+`;
+
+const ReviewContainer = styled.div`
+  border-top: solid 1px lightgray;
```

Comment:
May want to consider separating out some of these components into their own files. It's getting a bit long, so just a suggestion. Something you could refactor later.

Reply:
Perhaps.